sdk/database/helper/connutil/postgres.go

```diff
@@ -46,16 +46,16 @@ import (
 	"github.com/jackc/pgx/v4/stdlib"
 )
 
-// OpenPostgres parses the connection string and opens a connection to the database.
+// openPostgres parses the connection string and opens a connection to the database.
 //
 // If sslinline is set, strips the connection string of all ssl settings and
 // creates a TLS config based on the settings provided, then uses the
 // RegisterConnConfig function to create a new connection. This is necessary
 // because the pgx driver does not support the sslinline parameter and instead
 // expects to source ssl material from the file system.
 //
-// Deprecated: OpenPostgres will be removed in a future version of the Vault SDK.
-func OpenPostgres(driverName, connString string) (*sql.DB, error) {
+// Deprecated: openPostgres will be removed in a future version of the Vault SDK.
+func openPostgres(driverName, connString string) (*sql.DB, error) {
```

Comment:
This was only exported for tests but I decided we should not export this so the test usage was removed. This code path is tested because it is called indirectly by our tests.